components/derive/src/lib.rs

```diff
@@ -83,9 +83,12 @@ pub fn generate_protobuf_convert(input: TokenStream) -> TokenStream {
 /// Path to the module that was generated by the `exonum_build::protobuf_generate`
 /// and contains the original protobuf source files of the service.
 ///
-/// * `#[exonum(service_interface = "path")]`
+/// * `#[exonum(interfaces(default = "path", additional("Configure")))]`
```

Comment:
What do you think about to rename `Configure` to `Configurable` because interface ? Which - configurable.